include/gz/math/graph/Graph.hh

```diff
@@ -150,7 +150,7 @@ namespace graph
         {
           std::cerr << "[Graph::AddVertex()] The limit of vertices has been "
                     << "reached. Ignoring vertex." << std::endl;
-          return Vertex<V>::NullVertex;
+          return NullVertex<V>();
```

Comment:
this seems like an API change; we may need to do it, but let me check if sdformat will be affected

Reply:
yes, this will require changes in sdformat:

https://github.com/gazebosim/sdformat/blob/main/src/FrameSemantics.cc#L89

Reply:
I restored the original static members and added deprecations. It should be possible to keep using them.
[15b8975](https://github.com/gazebosim/gz-math/pull/606/commits/15b8975131bdef5f44c7932f4d120a15d21eca64)